src/chat_module/Client.java

```diff
@@ -1,9 +1,75 @@
-public class Client{
-    public static void main (String [] args) { 	
-        try {
-        	Client_Chat chatClient = new Client_Chat();
-        	chatClient.setVisible(true);
-        }
-        catch(ArrayIndexOutOfBoundsException e) { e.printStackTrace(); }
-    }	
+import java.net.*;
+import java.io.*;
+import java.util.*;
+import java.awt.*;
+import java.awt.event.*;
+import javax.swing.*;
+
+public class Client
```

Comment:
Created by MJ Constantino Jr.

Not in commit history due to integration issues